src/styles/blocks/services.scss

```diff
@@ -0,0 +1,152 @@
+@use '../utils/mixins' as *;
+
+.services {
+  @include page-grid;
+
+  padding: 120px 20px 48px;
+
+  @include for-tablet {
+    padding: 148px 41px 48px;
+
+    grid-template-rows: auto auto 51px 132px 162px 132px 162px 132px;
```

Comment:
Whoa, that's a weird value! 🤯 I think you don't have to do such complex calculations with rows. And just declare that it should be in x row. Then just tweak its position with margin.